packages/ListBoxBrowser/src/ListBoxBrowser.js

```diff
@@ -20,12 +20,7 @@ import Title from "./components/Title";
 
 const propTypes = {
   /**
-    An array of javascript objects holding the data structure for the ListBoxBrowser
-    The object shape must have at least a string label property and an array options property in one of the objects.
-    Also can hold any other kind of data for your own use.
-    ex.
-
-    [{ label: "One" }, { label: "Two", options: [{ label: "Three" }] }]
+    An array of javascript objects holding the data structure for the ListBoxBrowser. The object shape must have at least a string label property and an array options property in one of the objects. Also can hold any other kind of data for your own use.
```

Comment:
The part of the example and the data structure it's important maybe we should move into the `<!-- content -->` section on the `README.md` file?

Reply:
I will add that.